### PR TITLE
Praise Honkmother! 📯 🤡 (Fixes mime and clown crayons not changing colour) 

### LINKED ISSUE #7676: Broken Rainbow Crayon Color Change
**Problem Description**:
What is the problem?

The rainbow crayon wont change colors it stuck on yellow. Even after selecting a diff color it still stays on yellow.

**What did you expect to happen**:  For it to change to the color I selected.

**Why do you think this is a bug?**

It supposed to be able to change to diff colors.

**What happened instead**:  When selecting a diff color in the menu it stays on yellow

How is what happened different from what you expected?

It should change to the color selected like it supposed to. 

**Why is this bad/What are the consequences:**  Stuck with only yellow force to use normal crayons.

Why do you think this is an important issue?
Clowns need more colors. 

**Steps to reproduce the problem**: Get a rainbow crayon attempt to change it colors in color palette menu after using the crayon in your hand. After selecting any color the color box stays yellow and drawing anything get drawn in yellow still.

The most important section. Review everything you did leading up to causing the issue.
Walked around station for awhile. Lent the rainbow crayon to barber but he couldn't
get it to change colors so I tried and didn't work for me either.

**Possibly related stuff (which gamemode was it? cult What were you doing at the time? trying to change crayon color. Was anything else out of the ordinary happening?)**: Anything else you can tell us. nope just the rainbow crayon not changing colors.

### PR DESCRIPTION
**Pull request by Citinited**
Fixes #7676

:cl:
fix: Rainbow and mime crayons will now change colour as intended.
/:cl:

### CODE REVIEW

**Comment by Xantholne** [comment]
🤡🤡🤡🤡🤡🤡🤡🤡

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -190,7 +190,7 @@
 	..()
 
 /obj/item/toy/crayon/mime/Topic(href,href_list)
-	if((usr.restrained() || usr.stat || usr.is_in_active_hand(src)))
+	if(!(usr.is_in_active_hand(src)) || usr.restrained() || usr.stat)
```
Comment (Citinited) [inline code/game/objects/items/crayons.dm]: I don't even know who thought `if( usr.is_in_active_hand(src) )` returning was a good idea

File: code/game/objects/items/crayons.dm [hunk 2]
```diff
@@ -190,7 +190,7 @@
 	..()
 
 /obj/item/toy/crayon/mime/Topic(href,href_list)
-	if((usr.restrained() || usr.stat || usr.is_in_active_hand(src)))
```
Comment (Citinited) [inline code/game/objects/items/crayons.dm]: I don't even know who thought returning `if( usr.is_in_active_hand(src) )` was a good idea

Comment (Fox-McCloud) [inline code/game/objects/items/crayons.dm]: Just check for `if(incapacitated()) return`

Comment (Citinited) [inline code/game/objects/items/crayons.dm]: Eh, if they drop the crayon it doesn't make sense that they can change the crayon colour, right?

Comment (SamHPurp) [inline code/game/objects/items/crayons.dm]: as long as its still adjacent to them?

Comment (Fox-McCloud) [inline code/game/objects/items/crayons.dm]: yeah, checking for adjacency is better.

Comment (Citinited) [inline code/game/objects/items/crayons.dm]: @Fox-McCloud Done.

**Pull request merged**